priv/repo/migrations/20160328115654_create_relays.exs

```diff
@@ -7,7 +7,7 @@ defmodule Cog.Repo.Migrations.AddRelays do
       add :name, :text, null: false
       add :token_digest, :text, null: false
       add :enabled, :boolean, null: false
-      add :desc, :text, null: true
```

Comment:
Yeah, I wondered the same thing, but from looking at our discussions in the past, it seemed since we hadn't released this yet, that this would be OK. 

I'm cool with just adding a new migration, if that's preferred at this point.